sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/EditContactPage.java

```diff
@@ -117,4 +117,8 @@
   public static final By REMOVE_CASE_CTA_LINK = By.id("contactRemoveCase");
   public static final By CASE_CHANGE_POPUP_SUCCESS_MESSAGE =
       By.xpath("//*[contains(text(),'The source case of the contact has been changed')]");
+  public static final By CASE_CONFIRMED_RADIO_BUTTON =
```

Comment:
let's make this one generic, and create a method to click on one of those options (there is already in webdriverhelper a function clicking on item based on visible text, and the options from UI have a clear naming so we can pass them via bdd layer)